adafruit_vcnl4010.py

```diff
@@ -273,7 +278,7 @@ def ambient(self):
     # pylint: enable=inconsistent-return-statements
 
     @property
-    def ambient_lux(self):
+    def ambient_lux(self) -> int:
```

Comment:
This should be `float` based on the multiplication with a `float`.

Reply:
oops, should have checked the constant - sorry